hadoop-hdds/hadoop-dependency-client/README.md

```diff
@@ -35,20 +35,20 @@ mvn dependency:tree
 [INFO]
 [INFO] --- maven-dependency-plugin:3.0.2:tree (default-cli) @ hadoop-hdds-hadoop-dependency-client ---
 [INFO] org.apache.hadoop:hadoop-hdds-hadoop-dependency-client:jar:1.0.0
-[INFO] +- org.apache.hadoop:hadoop-annotations:jar:3.2.1:compile
+[INFO] +- org.apache.hadoop:hadoop-annotations:jar:3.2.2:compile
```

Comment:
This dependency tree is for Ozone 1.0.0, which still uses Hadoop 3.2.1.  I think we can keep it as is.  Alternatively, we might update the whole dependency tree (I'm sure there are other version upgrades since 1.0.0).  But I suggest doing that only right before 1.1.0 release.

Reply:
Have revert the changes.